package.json

```diff
@@ -6,10 +6,9 @@
   "scripts": {
     "json2ts": "./node_modules/.bin/json2ts -i src/modules/data/schemas/ -o src/modules/data/types/",
     "_test": "mocha -r ts-node/register tests/**/*.test.ts",
-    "_start": "nodemon --watch src --delay 1 --exec 'ts-node' src/main.ts",
     "test": "NODE_ENV=test npm-run-all --serial json2ts _test",
     "debug": "NODE_ENV=development npm-run-all --serial json2ts _test",
-    "start": "NODE_ENV=production npm-run-all --serial json2ts _start",
+    "start": "NODE_ENV=production nodemon -w src -i src/modules/data/types -d 1 -x ts-node src/main.ts",
```

Comment:
I changed to run `npm run json2ts` in `src/main.ts`.
The files in `src/modules/data/types` are changed.
And the service is restarted by nodemon.
In order to prevent the service from restarting, I made sure that the folder was ignored by the monitoring.